automatic/7zip.commandline/update.ps1

```diff
@@ -0,0 +1,37 @@
+. "$PSScriptRoot\..\7zip\update.ps1"
+
+$softwareNamePrefix = '7-zip'
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -FileNameBase '7zip'
+  $Latest.ChecksumType = 'sha256'
+
+  try {
+    $client = New-Object System.Net.WebClient
```

Comment:
this isn't .NET, so it don't matter if it's outside or inside the try/catch. (as far as I know)